README.md

```diff
@@ -2,18 +2,99 @@
 
 [![Build Status](https://travis-ci.org/nokia-wroclaw/innovative-project-jackdaw.svg?branch=master)](https://travis-ci.org/nokia-wroclaw/innovative-project-jackdaw)
 
-## Run
 
-`docker-compose up`
+## Table of contents
+1. Project goals
+2. Technologies
+3. Functionality of modules
+4. Installation
+5. Contributing
+
+
+## 1. Project goals
+The goal is to create PoC of platform for streaming calculation using Kafka and Docker in microservices architecture. This platform will allow to perform Big Data Calculation in Streaming mode. As a final product there will be created a web application responsible for data visualization.
+
+As an example of dataset we used one that represents Flights in Brasil from www.kaggle.com.
 
-or 
 
-`./run.sh`
+## 2. Technologies
+  * Java
+    * [Maven](http://maven.apache.org)
+  * [Docker](https://www.docker.com)
+  * JavaScript
+    * [Leaflet](https://leafletjs.com)
+    * [jQuery](https://jquery.com)
+    * [socket.io](https://socket.io)
+    * [node.js](https://nodejs.org)
+  * [Kafka](https://kafka.apache.org)
+    * [Confluent Schemat Registry](https://docs.confluent.io)
+  * [Travis Cl](https://travis-ci.org)
 
-## Stop
 
-`docker-compose down`
+## 3. Functionality of modules
+This project consist of three main modules:
+* Producer - reads data from file and sends messages using KafkaAvroSerializer to the 'FlightData' topic on Kafka
+* Consumer - reads data from 'FlightData' topic, uses KafkaAvroDeserializer, then converts data into a geoJSON and sends it to 'Visualizaion' topic on Kafka
+* Visualization
+	* consumer - reads data from 'Visualization' topic and constantly sends data with socket
```

Comment:
Indent with three spaces – not tabs.
Markdown interprets 3 spaces as an indentation.

Reply:
* `Visualization`